app/models/user.rb

```diff
@@ -0,0 +1,23 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise  :database_authenticatable, :registerable, :rememberable
+
+  has_one :profile, dependent: :destroy
+
```

Comment:
allow_nilするかユーザー作成時にprofileも作成するようにしましょう

profileがない時エラー出てしまいます。

<img width="1440" alt="Screen Shot 2022-07-05 at 18 10 58" src="https://user-images.githubusercontent.com/74767448/177293693-51ef5fce-1a31-4cf9-8431-7071d7d43e23.png">

Reply:
```suggestion
  # The path used after sign up.
  def after_sign_up_path_for(resource)
    new_profile_path
  end
```
一応、ここで最初にサインアップした後でプロフィール作成画面に飛ばしていました。
nilガードしておきます！